docs/building.md

```diff
@@ -12,29 +34,23 @@ for proper setup. If done correctly, you should have a GOROOT and GOPATH set in
 environment.
 
 If a Go development environment is setup, one can use `go get` to install the
-`registry` command from the current latest:
+`registry` source code from the current latest:
```

Comment:
Just saying that I think the change from "command" to "source code" is misleading. But it's fine with me if you want to fix it in a new PR.